.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -6,14 +6,34 @@ about: Something unexpected happen? Report an error, crash, or an incorrect beha
 
 Hello!
 
-Thank you for opening an issue.  Please note that we try to keep the Terraform MongoDB Atlas Provider issue tracker reserved for bug reports.  Please ensure you check open and closed issues first to ensure your issue hasn't already been reported (if it has been reported add a reaction, i.e. +1, to the issue).  
+Thank you for opening an issue. **These GitHub issues** are only for community reporting and assistance; as such, we don't have a guaranteed SLA.
 
-If your issue relates to Terraform itself, please open it in the Terraform repository https://github.com/hashicorp/terraform/issues.
+**If your issue relates to Terraform itself**, please open it in the Terraform repository https://github.com/hashicorp/terraform/issues.
 
-If you have an active MongoDB Atlas Support contract, the best way for us to assist you with the Terraform MongoDB Atlas Provider is through a [support ticket](https://support.mongodb.com/).
+**If you have an active MongoDB Atlas Support contract**, the best way for us to assist you with the Terraform MongoDB Atlas Provider is through a [support ticket](https://support.mongodb.com/).
+
+**Please note:** In order for us to provide the best experience in supporting our customers, we kindly ask to make sure that all the following sections are correctly filled with all the required information. Our support will **prioritise** issues that contain **all the required** information by using the **"one-click reproducible issues" principle**.
+
+**Please also note:** We try to keep the Terraform MongoDB Atlas Provider issue tracker **reserved for bug reports**. Please ensure you **check open and closed issues first** to ensure your issue hasn't already been reported (if it has been reported add a reaction, i.e. +1, to the issue).
+
+### Guidelines
+
+In order to follow the **"one-click reproducible issues" principle**, please follow these guidelines:
+
+* We should be able to make no changes to your provided script and **be able to run `terraform plan` and `terraform apply` with no errors**.
```

Comment:
```suggestion
* We should be able to make no changes to your provided script and **be able to run a local execution reproducing the issue**.
```
Must be able to run with no errors was a little confusing to me as multiple bugs are related to non empty plans or errors in apply